packages/utils/upgrade/resources/codemods/5.0.0/sqlite3-to-better-sqlite3.code.ts

```diff
@@ -0,0 +1,43 @@
+import type { StringLiteral, Transform } from 'jscodeshift';
+
+const transform: Transform = (file, api) => {
+  const { j } = api;
+
+  // Parse the file content
+  const root = j(file.source);
+
+  if (!file.path.includes('config/database.js')) {
+    return file.source;
+  }
+  const targetProperties = new Set([
+    'sqlite3',
+    'vscode/sqlite3',
+    'sqlite-legacy',
+    'better-sqlite3',
+  ]);
+
+  return root
+    .find(j.ObjectExpression)
+    .forEach((path) => {
+      j(path)
+        .find(j.Property)
+        .forEach((propertyPath) => {
+          // Check if the property name is one of the targets
+          const key = propertyPath.node.key;
+          const propertyName = key.type === 'Identifier' ? key.name : (key as StringLiteral).value;
+
+          if (targetProperties.has(propertyName)) {
+            // Rename the property to 'sqlite'
+            if (key.type === 'Identifier') {
+              key.name = 'sqlite';
+            } else {
+              // For Literal types
+              (key as StringLiteral).value = 'sqlite';
+            }
+          }
```

Comment:
Just a suggestion, but you can achieve better type narrowing using type guards from jscodeshift utils
```suggestion
          const key = propertyPath.node.key;

          // If the key is an identifier, and its key is in targetProperties, rename it to 'sqlite'
          if (j.Identifier.check(key) && targetProperties.has(key.name)) {
            key.name = 'sqlite';
          }

          // If the key is a string literal, and its value is in targetProperties, rename it to 'sqlite'
          if (j.StringLiteral.check(key) && targetProperties.has(key.value)) {
            key.value = 'sqlite';
          }
```

Reply:
Also, I'm curious as in which scenario sqlite can be found in either a key or a string literal within object properties? Could you add an example/a description in the codemod file about what it's doing? That would be very helpful to understand what's happening without context :pray:

Reply:
Yeah, I don't think we need to rename the key here, just the value. 

The goal here is that a database config 'client' value for sqlite must now be 'sqlite' and not a specific package name.

A comment for exactly what the breaking change is and how it's being addressed would be great, especially to set a good example as the very first codemod 😆